hassio/src/addon-view/config/hassio-addon-config.ts

```diff
@@ -204,8 +202,12 @@ class HassioAddonConfig extends LitElement {
 
     try {
       await setHassioAddonOption(this.hass, this.addon.slug, {
-        options: this._options!,
+        options:
+          !this._yamlMode && this._canShowSchema && this.addon.schema
```

Comment:
We should be able to just check `_yamlMode`?

Reply:
True!